Source/EventFlow.Autofac/Registrations/AutofacServiceRegistration.cs

```diff
@@ -193,12 +201,11 @@ public void Decorate<TService>(Func<IResolverContext, TService, TService> factor
         public IRootResolver CreateResolver(bool validateRegistrations)
         {
             var container = _containerBuilder.Build();
-            var autofacRootResolver = new AutofacRootResolver(container);
-
-            if (validateRegistrations)
-            {
-                autofacRootResolver.ValidateRegistrations();
-            }
+            using (var autofacRootResolver = new AutofacRootResolver(container))
```

Comment:
Is it possible disposed exceptions to be because of this using?

Reply:
Assume the container gets disposed with it,  not sure why there's a `return new AutofacRootResolver(container);`. 
 Should remove the `using `and have `return autofacRootResolver`